### PR TITLE
1.1.8 Patch Release

### PR DESCRIPTION
**Pull request by jfy133**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/taxprofiler/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/taxprofiler _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nf-test test main.nf.test -profile test,docker`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,30 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.1.8dev - Augmented Akita Patch [2024-06-20]
```
Comment (ramprasadn) [inline CHANGELOG.md:6]:
```suggestion
## v1.1.8 - Augmented Akita Patch [2024-06-20]
```

**Review by ramprasadn — APPROVED** [review]
Nice work! 👍🏻 
Just one minor comment!

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,30 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.1.8 - Augmented Akita Patch [2024-06-20]
+
+### `Added`
+
+- [#487](https://github.com/nf-core/taxprofiler/pull/487) Updated to nf-core pipeline template v2.14.1 (added by jfy133)
```
Comment (FriederikeHanssen) [inline CHANGELOG.md:10]:
```suggestion
- [#487](https://github.com/nf-core/taxprofiler/pull/487) Updated to nf-core pipeline template v2.14.1 (added by @jfy133)
```

File: modules/nf-core/kmcp/profile/main.nf [hunk 3]
```diff
@@ -2,15 +2,14 @@ process KMCP_PROFILE {
     tag "$meta.id"
     label 'process_medium'
 
-    conda "bioconda::kmcp=0.9.1"
+    conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/kmcp:0.9.1--h9ee0642_0':
-        'biocontainers/kmcp:0.9.1--h9ee0642_0' }"
+        'https://depot.galaxyproject.org/singularity/kmcp:0.9.4--h9ee0642_0':
+        'biocontainers/kmcp:0.9.4--h9ee0642_0' }"
```
Comment (FriederikeHanssen) [inline modules/nf-core/kmcp/profile/main.nf:8]: are you tracking version updates of tools in the changelog normally?

**Review by FriederikeHanssen — APPROVED** [review]

File: modules/nf-core/kmcp/profile/main.nf (hunk 3 above)
Comment (jfy133) [reply modules/nf-core/kmcp/profile/main.nf:8]: Yes, in the table under 'depdnecnies'. Borrowed it from mag as I thought it was a nice idea :)

**Pull request merged**